examples/coco/README.md

```diff
@@ -1,52 +1,165 @@
+
 # README
 
-This describes the coco integration with TL. We use `coco 14` training and validation files combined with a `monilenet_v2` backbone and a `pix2pix` based decoder. 
+## Semantic Segmantation - COCO 
+
+
+In this example, we show the use of Tensorleap on a Computer Vision task - Semantic Segmentation on COCO data. We use `coco 14` training and validation data files combined with a `mobilenet_v2` backbone and a `pix2pix` based decoder. 
+
+### The Task
+
+The goal of Semantic Segmentation is to label each pixel of an image with a corresponding class. Because the prediction is for every pixel in the image, this task is commonly referred to as dense prediction. In Semantic Segmentation, the boundary of objects is labeled with a mask, and object classes are labeled with a class label. The output itself is a high-resolution image (typically of the same size as the input image) in which each pixel is classified to a particular class. 
+
+### The Dataset
+
+COCO (Common Objects in Context) is large-scale object detection, segmentation, and captioning dataset. The COCO Dataset has 80 classes.
+
+### The Model
+
+#### Model Introduction
+
+We use a modified U-Net, which is an end-to-end Fully Convolutional Network Model for the task. The U-Net was developed by Olaf Ronneberger et al. for Biomedical Image Segmentation. The architecture consists of an encoder - which is used to capture the image context, and a decoder - which is used to enable precise localization using transposed convolutions. 
+
+In the original paper, the U-Net is described as follows:
+
+<img alt="Unet" height="400" src="./coco/images/Unet.png" width="600"/>
```

Comment:
Use an image from the platform here

Reply:
@yotamazriel model's too big to captured well by image.
![image](https://user-images.githubusercontent.com/95576526/160556625-5c372b88-393c-45d5-8437-27f58b87ff54.png)